api/src/main/java/org/pmiops/workbench/actionaudit/auditors/LeonardoRuntimeAuditor.java

```diff
@@ -0,0 +1,7 @@
+package org.pmiops.workbench.actionaudit.auditors;
+
+import java.util.List;
+
+public interface LeonardoRuntimeAuditor {
```

Comment:
Nit - some of the Runtime classes are prefixed with `Leonardo` while others are not. Is there a pattern being followed?